rollup.config.js

```diff
@@ -1,8 +1,12 @@
 import babel from "@rollup/plugin-babel";
 import { terser } from "rollup-plugin-terser";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
```

Comment:
commonjs is needed because I think it looks at the `react` lib even though it's marked as external and then dies because it's cjs and rollup only supports esm by default.